src/current/molt/molt-fetch.md

```diff
@@ -37,7 +37,17 @@ Complete the following items before using MOLT Fetch:
 
 	- For PostgreSQL sources, enable logical replication. In `postgresql.conf` or in the SQL shell, set [`wal_level`](https://www.postgresql.org/docs/current/runtime-config-wal.html) to `logical`.
 
-	- For MySQL sources, enable [GTID](https://dev.mysql.com/doc/refman/8.0/en/replication-options-gtids.html) consistency. In `mysql.cnf`, in the SQL shell, or as flags in the `mysql` start command, set `gtid-mode` and `enforce-gtid-consistency` to `ON` and set `binlog_row_metadata` to `full`.
+	- For MySQL **8.0 and later** sources, enable [GTID](https://dev.mysql.com/doc/refman/8.0/en/replication-options-gtids.html) consistency. Set the following values in `mysql.cnf`, in the SQL shell, or as flags in the `mysql` start command:
+		- `--enforce-gtid-consistency=ON`
+		- `--gtid-mode=ON`
+		- `--binlog-row-metadata=full`
+
+	- For MySQL **5.7** sources, set the following values. Note that `binlog-row-image` is used instead of `binlog-row-metadata`. Set `server-id` to a unique integer that differs from any other MySQL server you have in your cluster (e.g., `3`).
```

Comment:
Did you want this line starting with "For MySQL 5.7..." to be indented the same level as "For MySQL 8.0"...? Right now, the 5.7 bullet is rendering inline with the flags.